recipes/pytest-localstack/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pytest-localstack" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 91be109a98f3737e10b0b1de8f0cf48f30abee878a77099862fad2bed6de689c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - botocore >=1.4.31,!=1.4.45
```

Comment:
Are all these host dependencies needed? (Are they needed to install the package?)

Reply:
@CJ-Wright they were included in the skeleton, but you're correct- I removed them and they weren't necessary to install. thanks!